.github/workflows/integ-test-inject-develocity.yml

```diff
@@ -118,10 +127,21 @@ jobs:
     env:
       DEVELOCITY_INJECTION_ENABLED: true
       DEVELOCITY_URL: 'https://localhost:3333/'
-      DEVELOCITY_PLUGIN_VERSION: 3.17.3
+      DEVELOCITY_PLUGIN_VERSION: ${{ matrix.plugin-version }}
       DEVELOCITY_CCUD_PLUGIN_VERSION: '2.0'
       # Access key also set as an env var, we want to check it does not leak
-      DEVELOCITY_ACCESS_KEY: ${{ secrets.DEVELOCITY_ACCESS_KEY }}
+      ${{matrix.accessKeyEnv}}: ${{ secrets.DEVELOCITY_ACCESS_KEY }}
```

Comment:
For this test, we probably want to set _both_ `DEVELOCITY_ACCESS_KEY` and `GRADLE_ENTERPRISE_ACCESS_KEY`, so that we can confirm that both are cleared by the implementation.

So again, the `accessKeyEnv` isn't required.

Reply:
right, that's a better test, will change to setting both env vars